app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def full_title page_title = ""
+	  base_title = I18n.t("title")
+	  page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+  end
```

Comment:
sai indent, 2 space = 1 tab
```suggestion
  def full_title page_title = ""
    base_title = I18n.t("title")
    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
  end
```